autogen/agentchat/contrib/gpt_assistant_agent.py

```diff
@@ -22,9 +22,11 @@ class GPTAssistantAgent(ConversableAgent):
     def __init__(
         self,
         name="GPT Assistant",
+        is_termination_msg: Optional[Callable[[Dict], bool]] = None,
```

Comment:
Is this necessary if `kwargs` is given? Code below passes `kwargs` to `super()` which already handles setting the `is_termination_msg` attribute? See: https://github.com/microsoft/autogen/blob/main/autogen/agentchat/conversable_agent.py#L105

Reply:
Consistency is important. We should keep the same default termination behavior across assistants and give users option to change it :) I would cut this part-- its not essential to your PR.